### PR TITLE
✨ Version bumped amp-experiment to 1.0

### PR DESCRIPTION
**Pull request by torch2424**
This starts the new [amp-experiment](https://github.com/ampproject/amphtml/issues/20225) by version bumping this to 1.0 😄 

# Example

screen shot 2019-02-27 at 2 40 45 pm

### CODE REVIEW

**Comment by torch2424** [comment]
@jridgewell Discussed offline with @lannka , they don't see this as a blocker currently, as we are bumping major versions, and shouldn't be a real problem for current amp-experiment pages until we launch into PROD.

cc @lannka for future discussion 😄

**Comment by aghassemi** [comment]
@torch2424 @jridgewell @lannka could someone point me to the code where building `-latest.js` differentiates between major and minor version bumps?

**Comment by torch2424** [comment]
This PR now requires #21193 and #21190 to be merged first.

**Comment by rsimha** [comment]
The race between versions to be the latest has been fixed by 
#21190. See https://github.com/ampproject/amphtml/pull/21190#issue-257273690 for more info.

File: bundles.config.js [hunk 1]
```diff
@@ -131,7 +131,7 @@ exports.extensionBundles = [
   {name: 'amp-google-document-embed', version: '0.1', type: TYPES.MISC},
   {name: 'amp-dynamic-css-classes', version: '0.1', type: TYPES.MISC},
   {name: 'amp-embedly-card', version: '0.1', type: TYPES.MISC},
-  {name: 'amp-experiment', version: '0.1', type: TYPES.MISC},
+  {name: 'amp-experiment', version: ['0.1', '1.0'], type: TYPES.MISC},
```
Comment (rsimha) [inline bundles.config.js]: This will now require `latestVersion: '1.0'` in addition to the new version.

Comment (torch2424) [reply bundles.config.js]: Thank you! 😄

**Comment by torch2424** [comment]
Fixed changes from #21193 , and the `-latest` brought up by @aghassemi and @rsimha 

This is now ready for review @zhouyx m and anyone else who is interested 😄

File: bundles.config.js [hunk 2]
```diff
@@ -325,8 +325,8 @@ exports.extensionBundles = [
   },
   {
     name: 'amp-experiment',
-    version: '0.1',
-    latestVersion: '0.1',
+    version: ['0.1', '1.0'],
+    latestVersion: '1.0',
```
Comment (jridgewell) [inline bundles.config.js]: Latest version should be 0.1, 1.0 is still experimental.

Comment (rsimha) [reply bundles.config.js]: (I generally agree with this comment.) I noticed that `amp-story`, `amp-mustache`, and `amp-access-laterpay` all list the newest (experimental?) version as `latestVersion`. Should they too be rolled back until the newest version is ready for wide release?

https://github.com/ampproject/amphtml/blob/2b8b439203444545acd5c8780ae5ea79ac972b0a/bundles.config.js#L636-L638
https://github.com/ampproject/amphtml/blob/2b8b439203444545acd5c8780ae5ea79ac972b0a/bundles.config.js#L522-L524
https://github.com/ampproject/amphtml/blob/2b8b439203444545acd5c8780ae5ea79ac972b0a/bundles.config.js#L52-L54

**Comment by torch2424** [comment]
@jridgewell Made requested changes 😄 

Issues mentioned by @rsimha were moved to #21172 👍

**Pull request merged**